packages/fast-colors/src/parse-color.ts

```diff
@@ -344,12 +193,21 @@ export function parseColorWebRGBA(raw: string): ColorRGBA64 | null {
 }
 
 // Expects any of the CSS color names https://www.w3schools.com/colors/colors_names.asp

```

Comment:
nit: not part of your changes but can we change the link to https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords